html/index.html

```diff
@@ -0,0 +1,190 @@
+                worship!
+              </p>
+            </div>
+            <div class="card flex">
+              <i class="fa-solid fa-person-praying"></i>
+              <h2 class="tag">Prayer</h2>
+              <p class="tag-info">
+                Anointed prayers session with spiritfilled worship by anointed
+                ministers of the Gospel; come and experience the intimacy in
+                worship!
+              </p>
+            </div>
+            <div class="card flex">
+              <i class="fa-brands fa-superpowers"></i>
+              <h2 class="tag">Impartation</h2>
+              <p class="tag-info">
+                Anointed impartation session with spiritfilled worship by
+                anointed ministers of the Gospel; come and receive empowerment
+                of the Holy Spirit!
+              </p>
+            </div>
+            <div class="card flex">
+              <i class="fa-solid fa-person-chalkboard"></i>
+              <h2 class="tag">Teaching</h2>
+              <p class="tag-info">
+                Anointed teaching session with spiritfilled worship by anointed
+                ministers of the Gospel; come and experience the intimacy in
+                worship!
+              </p>
+            </div>
+            <div class="card flex">
+              <i class="fa-solid fa-fire-flame-curved"></i>
+              <h2 class="tag">Baptism</h2>
+              <p class="tag-info">
+                Anointed baptism session with spiritfilled worship by anointed
+                ministers of the Gospel; come and experience of the baptism of
+                Fire!
+              </p>
+            </div>
+          </div>
+          <a href="#" class="see-prog">SEE THE WHOLE PROGRAM</a>
+          <a href="#" class="mobile">Join The Kingdom Comes Global Worship</a>
+        </div>
+      </section>
+      <!--===== Dynamic content with JS start here ===== -->
+      <section class="speakers flex">
+        <div class="caption flex">
+          <h2>Featured Speakers</h2>
+        </div>
+        <div class="feat-speaker flex"></div>
+        <button class="flex" id="show-more">Show More</button>
+      </section>
+      <!--===== Dynamic content with JS end here ===== -->
+
+      <section class="partner" id="Sponsor">
+        <div class="caption flex">
+          <h2>Partners</h2>
+          <div class="featured"></div>
+        </div>
+        <div class="partner-logo flex">
+          <img
+            class="sponsors"
+            src="/images/ssuc-logo.png"
+            alt="Partner logo"
+          />
+          <img class="sponsors" src="/images/ECS-LOGO.png" alt="Partner logo" />
+          <img
+            class="sponsors"
+            src="/images/SSCC-LOGO.png"
+            alt="Partner logo"
+          />
+          <img
+            class="sponsors"
+            src="/images/Winners-Logo.png"
+            alt="Partner logo"
+          />
+        </div>
```

Comment:
- [ ] In the desktop version you have the footer but I think the elements  are not align beside each others but in your design they are different so please make them to be beside each other

- [ ] Kindly make the width and the height for the images smaller like the original design because it is important to stick with the design style so kindly fix it ✅✅🙏🏻🙏🏻

- your design
![image](https://user-images.githubusercontent.com/58553711/182903897-cf4ea950-68e5-459c-b1a4-89da5b3cb66c.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/182904046-ecbea7d5-0053-4210-bb3e-49b5d73adf87.png)

Reply:
* Kindly try to stick to the design as the previous CR mentioned above